Download all .fastq files in a project/Download all .fastq files in a project.ipynb

```diff
@@ -0,0 +1,6347 @@
+{
```

Comment:
<p>Nitpick: change "in each bundle" to "in a given bundle" (since the code doesn't actually do it for every bundle)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCellAtlas/data-consumer-vignettes/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>